packages/bpk-component-navigation-bar/src/BpkNavigationBar.js

```diff
@@ -29,22 +29,30 @@ import STYLES from './BpkNavigationBar.module.scss';
 
 const getClassNames = cssModules(STYLES);
 
+export const BAR_STYLES = {
+  'default': 'default',
+  onDark: 'on-dark',
+};
+export type BarStyle = $Values<BAR_STYLES>
```

Comment:
Do you think you could convert it into TS as we are moving away from flow? We've added this bit in the contribution guidelines https://github.com/Skyscanner/backpack/blob/main/CONTRIBUTING.md#typescript but maybe it's not too visible 😢

Reply:
Migrated to typescript, does this mean it will need a major version change or still a minor?

Reply:
I think it's ok to keep it as minor 🤔 Looking at the changelog, I see we've generally released these as minor.